apps/frontend/src/components/ViewFormsList/ViewFormsList.tsx

```diff
@@ -14,18 +14,72 @@ import {
   Thead,
   Tr,
   Link,
+  Input,
 } from '@chakra-ui/react';
 import { getAllForms } from '../../utils/sendRequest';
 import { FormData } from '../../types/formData';
+import { SortOptions } from '../../enums/SortOrder';
 
 export default function ViewFormsList() {
-  const [forms, setForms] = useState<FormData[] | null>(null);
+  const [forms, setForms] = useState<FormData[]>([]);
+  const [allForms, setAllForms] = useState<FormData[]>([]); // this is used to get all forms again after removing a filter/search term
+  const [sortBy, setSortBy] = useState<SortOptions>(SortOptions.NAME);
+  const [searchTerm, setSearchTerm] = useState<string>('');
 
   const getForms = async () => {
     const allForms = await getAllForms();
     setForms(allForms);
+    setAllForms(allForms);
   };
 
+  // sort forms
+  useMemo(() => {
```

Comment:
I think these two methods should be useEffects rather than useMemos. useMemo is used to memoize a value, and is used to useEffect perform some actions when certain state variables change. It won't affect the functionality, but since you aren't returning/storing any values here, I think useEffect is a better choice.

Reply:
oh cool ty for the info!!